docs/_docs/Starters Academy (LDSSA)/05-Capstone-Project.md

```diff
@@ -120,15 +120,16 @@ The capstone will automatically be considered a fail if:
 
 ### Calendar
 
-Description |  Date | 
--------------|-------------|
-Kick off |  17th January 2022 | 
-Trial round of requests |  4th or 5th February 2022 |  
-Deadline report 1 and app launched |  6th February 2022, 23h59 UTC  |
-First round of requests  |  7th February 2022 at 00h01UTC to 14th February 2022 at 23h59 UTC  |
-Comments to report 1 made by instructors  |  14th February 2022 23h59 UTC  |
-Deadline report 2 + redeploy + address comments to report 1 |  6th March 2022, 23h59 UTC  |
-Second round of requests |  7th March 2022 at 00h01UTC to 14th March 2022 at 23h59 UTC  |
-Comments to report 2 made by instructors  |  13th March 2022, 23h59 UTC  |
-Deadline address comments to report 2  |  20th March 2022, 23h59 UTC  |
-Graduates announced  |  31st March 2022  |
+| Description                                                 | Date                                                             |
```

Comment:
Same suggestion: move this to a Google Calendar and leave here the link. 

It will be quite hard to maintain both the Cal and the Wiki updated and synced as time goes by.